dags/roger/core.py

```diff
@@ -28,10 +29,24 @@
 from roger.roger_db import RedisGraph
 from string import Template
 from urllib.request import urlretrieve
+from itertools import chain
 
 log = get_logger ()
 config = get_config ()
 
+def jsonl_iter(file_name):
```

Comment:
can this two functions be part of Util class here in core.py?

Reply:
Sure.  I was also thinking of adding them to kg_utils, but adding them to the Util class makes at least as much sense.